command_prompt.bash

```diff
@@ -199,7 +201,14 @@ _seafly_command_prompt() {
     # Normal prompt indicates that the last command ran successfully.
     # Alert prompt indicates that the last command failed.
     _seafly_colors=("$SEAFLY_ALERT_COLOR" "$SEAFLY_NORMAL_COLOR")
-    local prompt_end="\[\${_seafly_colors[\$((!\$?))]}\] $SEAFLY_PROMPT_SYMBOL\[\$NOCOLOR\] "
+    local prompt_end
+    if [[ $SEAFLY_MULTILINE = 1 ]]; then
+        prompt_start="\n$prompt_start"
```

Comment:
The `prompt_start` change on line 206 is better placed up where prompt_start is set, just after all the `SHOW_USER` & `SHOW_HOST` logic is done.

```sh
if [[ $SEAFLY_MULTILINE = 1 ]]; then
    prompt_start="\n$prompt_start"
fi
```

That way all the `prompt_start`, `prompt_middle` and `prompt_end` logic is clustered together.